client/src/components/Menu/index.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable consistent-return */
```

Comment:
This page should be on the Pages directory and keep the components on the components directory.